emperor/support_files/vendor/js/three.js-plugins/SVGRenderer.js

```diff
@@ -159,7 +159,15 @@ THREE.SVGRenderer = function () {
 				_v1 = element;
 				_v1.x *= _svgWidthHalf; _v1.y *= - _svgHeightHalf;
 
-				renderSprite( _v1, element, material );
+				if ('text' in element.object) {
```

Comment:
Would you mind making a note of these changes at the top of the file? Just to keep track of this.